apps/web/components/ActionCategoryCard/ActionCategoryCard.tsx

```diff
@@ -0,0 +1,323 @@
+import { Icon as IconType } from '../IconRC/iconMap'
+import { LinkV2 } from '../Link/LinkV2'
+
+export const STACK_WIDTH = 280
+
+export type CategoryCardTag = {
+  label: string
+  href?: string
+  onClick?: () => void
+  disabled?: boolean
+  outlined?: boolean
+}
+
+export type CTAProps = {
+  label: string
+  /** Allows for simple variant configuration of the button. If buttonType is defined it will supersede this property. */
+  variant?: ButtonTypes['variant']
+  /** Allows for full buttonType control. Supersedes the variant property when both are defined. */
+  buttonType?: ButtonTypes
+  size?: ButtonSizes
+  icon?: IconType
+  iconType?: 'filled' | 'outline'
+  onClick?: () => void
+  disabled?: boolean
+}
+
+type SidePanelItems = {
+  icon?: React.ReactNode
+  title: string
+}
+
+type SidePanelConfigProps = {
+  buttonLabel?: string
+  items?: Array<SidePanelItems>
+  cta?: CTAProps
+}
+
+export type ActionCategoryCardProps = {
+  ref?: UseMeasureRef<HTMLElement>
+  width?: number
+  icon?: React.ReactElement
+  heading: string
+  headingAs?: TextProps['as']
+  headingVariant?: TextProps['variant']
+  text: string
+  tags?: CategoryCardTag[]
+  tagOptions?: Pick<TagProps, 'hyphenate' | 'truncate' | 'textLeft'>
+  href?: string
+  colorScheme?: 'blue' | 'purple' | 'red'
+  /** The heading above is truncated instead of overflowing */
+  truncateHeading?: TextProps['truncate']
+  /** Use event listener to check wether or not to place the image below the text content */
+  autoStack?: boolean
+  /** The card width breakpoint that the image should stack below content when autoStack = true */
+  stackWidth?: number
+  /** Hyphenate the heading */
+  hyphenate?: boolean
+  to?: string
+
+  //TODO: DOCUMENT THIS
+  sidePanelConfig?: SidePanelConfigProps
+  customBottomContent?: React.ReactNode
+}
+
+const colorSchemes = {
+  blue: {
+    textColor: 'blue400',
+    borderColor: 'blue200',
+    tagVariant: 'blue',
+  },
+  purple: {
+    textColor: 'purple400',
+    borderColor: 'purple200',
+    tagVariant: 'purple',
+  },
+  red: {
+    textColor: 'red600',
+    borderColor: 'red200',
+    tagVariant: 'red',
+  },
+} as const
+
+const Component = forwardRef<HTMLElement, ActionCategoryCardProps>(
+  (
+    {
+      width,
+      stackWidth = STACK_WIDTH,
+      heading,
+      headingAs = 'h3',
+      headingVariant = 'h3',
+      icon,
+      text,
+      href = '/',
+      tags = [],
+      colorScheme = 'blue',
+      truncateHeading = false,
+      sidePanelConfig,
+      hyphenate = false,
+      tagOptions,
+      autoStack,
+      customBottomContent,
+      ...rest
+    },
+    ref,
+  ) => {
+    const { borderColor, textColor, tagVariant } = colorSchemes[colorScheme]
+
+    const hasTags = Array.isArray(tags) && tags.length > 0
+    const shouldStack = width && width < stackWidth
+
+    const renderCTA = () => {
+      const hasCTA = !!sidePanelConfig?.cta?.label
+      const cta = sidePanelConfig?.cta
```

Comment:
your `hasCTA` check covers both cta and cta label, so i think you dont need the two :)